packages/jni-swig-stub/src/main/jni/realm_api_helpers.cpp

```diff
@@ -228,6 +228,7 @@ void register_login_cb(realm_app_t *app, realm_app_credentials_t *credentials, j
             credentials,
             // FIXME Refactor into generic handling of network requests, like
             //  https://github.com/realm/realm-java/blob/master/realm/realm-library/src/main/cpp/io_realm_internal_objectstore_OsApp.cpp#L192
+            //  see 'app_register_email_password'
```

Comment:
This is generalized in https://github.com/realm/realm-kotlin/pull/517/files#diff-b191a2b8280990335be6159a3281434b292df6d13e86ce0e05ff0c55085e1161R46